diff -Naur a/data/org.gnome.Fractal.desktop.in.in b/data/org.gnome.Fractal.desktop.in.in
--- a/data/org.gnome.Fractal.desktop.in.in	2023-07-07 02:08:03.548201616 +0530
+++ b/data/org.gnome.Fractal.desktop.in.in	2023-07-07 02:08:53.424233400 +0530
@@ -4,7 +4,7 @@
 GenericName=Fractal group messaging
 Exec=fractal
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
-Icon=@icon@
+Icon=snap.fractal.@icon@
 Type=Application
 StartupNotify=true
 X-GNOME-UsesNotifications=true
diff -Naur a/data/org.gnome.Fractal.metainfo.xml.in.in b/data/org.gnome.Fractal.metainfo.xml.in.in
--- a/data/org.gnome.Fractal.metainfo.xml.in.in	2023-07-07 02:08:23.268214007 +0530
+++ b/data/org.gnome.Fractal.metainfo.xml.in.in	2023-07-07 02:09:08.384243207 +0530
@@ -3,6 +3,7 @@
 <!-- Copyright 2022 Julian Sparber <julian@sparber.net> -->
 <component type="desktop-application">
   <id>@app-id@</id>
+  <icon type="stock">snap.fractal.@app-id@</icon>
   <launchable type="desktop-id">@app-id@.desktop</launchable>
   <name>Fractal</name>
   <summary>Matrix group messaging app</summary>
@@ -44,7 +45,7 @@
     <value key="Purism::form_factor">workstation</value>
     <value key="Purism::form_factor">mobile</value>
   </custom>
-  
+
   <content_rating type="oars-1.1">
     <content_attribute id="social-chat">intense</content_attribute>
     <content_attribute id="social-audio">intense</content_attribute>
@@ -56,7 +57,7 @@
         <p>Fractal 5.beta1 is the first beta release since the rewrite of
           Fractal to take advantage of GTK 4 and the Matrix Rust SDK. It is the
           result of over two years of work.</p>
-        
+
         <p>New features since Fractal 5.alpha1:</p>
         <ul>
           <li>Joining room by ID, by alias or with a Matrix URI</li>
@@ -74,10 +75,10 @@
         <p>Of course, there are a also a lot of less visible changes, fixes and
           translations thanks to all our contributors, and our upstream
           projects.</p>
-        
+
         <p>As the version implies, this is still considered beta stage and might
           trigger crashes or other bugs but overall should be pretty stable.</p>
-        
+
         <p>A list of blocking issues for the release of version 5 can be found
           in the Fractal v5 milestone on GitLab.</p>
       </description>
